src/city.json

```diff
@@ -0,0 +1,286 @@
+{
```

Comment:
Can you try importing this JSON in your app and using it instead of the hardcoded data? 😃 
First, decide where to import it - which component(s) need access to the data?
Then, console.log it to make sure it's there and looks as expected. 
Finally, decide how to pass the relevant bits of data down to the right components... 

Have a go first, and we can look at what you've done if you need help.